interface/skin.dmf

```diff
@@ -1189,8 +1189,8 @@ window "mainwindow"
 		type = MAIN
 		pos = 281,0
 		size = 640x440
-		anchor1 = none
-		anchor2 = none

```

Comment:
All this stuff appears to just be me opening the skin.dmf in 514 and it *having its way* with it.